core/src/components/modal/modal.scss

```diff
@@ -180,23 +180,19 @@ $modals: (
       overflow-y: auto;
     }
 
-    slot[name='actions'] {
+    slot[name='bottom'] {
```

Comment:
If the Modal body text is long it cuts off at the bottom, hiding the action buttons. Is this something we should address or is it up to the user to adjust padding as needed?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/8b920bc4-1232-461b-a3cb-3c580f8fe781)